src/System Application/App/Extension Management/src/ExtensionInstallationImpl.Codeunit.al

```diff
@@ -241,6 +241,18 @@ codeunit 2500 "Extension Installation Impl"
         exit(true);
     end;
 
+    local procedure UninstallExtensionSilentlyByAppId(AppId: Guid; ClearData: Boolean; ClearSchema: Boolean): Boolean
+    begin
+        CheckPermissions();
+        AssertIsInitialized();
+        DotNetNavAppALInstaller.ALUninstallNavAppByAppId(AppId, ClearData, ClearSchema);
+
+        if IsInstalledByAppId(AppId) then
+            exit(false);
+
+        exit(true);
+    end;
+
     procedure UninstallExtensionWithConfirmDialog(PackageId: Guid; ClearData: Boolean; ClearSchema: Boolean): Boolean
```

Comment:
Do we need the App ID on these ones too? Or this is not a problem since installed apps are not from BC14 and older?